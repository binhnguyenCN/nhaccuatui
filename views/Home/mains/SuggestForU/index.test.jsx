// libs
import renderer from 'react-test-renderer';
// layouts
import SuggestForYou from '.';

describe('render suggest for you', () => {
  it('should renders correctly', () => {
    // act
    const tree = renderer.create(<SuggestForYou />).toJSON();
    // assert
    expect(tree).toMatchSnapshot();
  });
});
