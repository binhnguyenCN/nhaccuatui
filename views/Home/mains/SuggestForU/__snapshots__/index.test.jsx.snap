// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render suggest for you renders correctly 1`] = `
<div
  className="suggest-foru-wrapper"
>
  <div
    className="suggest-foru-inner"
  >
    <h3
      className="suggest-title"
    >
      GỢI Ý DÀNH CHO BẠN
    </h3>
    <p
      className="suggest-description"
    >
      Thưởng thức những ca khúc phù hợp nhất với bạn
    </p>
    <div
      className="suggest-btn"
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <span
        aria-label="caret-right"
        className="anticon anticon-caret-right"
        color="#2c83ed"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="caret-right"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="0 0 1024 1024"
          width="1em"
        >
          <path
            d="M715.8 493.5L335 165.1c-14.2-12.2-35-1.2-35 18.5v656.8c0 19.7 20.8 30.7 35 18.5l380.8-328.4c10.9-9.4 10.9-27.6 0-37z"
          />
        </svg>
      </span>
      Nghe bài hát
    </div>
  </div>
</div>
`;

exports[`render suggest for you should renders correctly 1`] = `
<div
  className="suggest-foru-wrapper"
>
  <div
    className="suggest-foru-inner"
  >
    <h3
      className="suggest-title"
    >
      GỢI Ý DÀNH CHO BẠN
    </h3>
    <p
      className="suggest-description"
    >
      Thưởng thức những ca khúc phù hợp nhất với bạn
    </p>
    <div
      className="suggest-btn"
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <span
        aria-label="caret-right"
        className="anticon anticon-caret-right"
        color="#2c83ed"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="caret-right"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="0 0 1024 1024"
          width="1em"
        >
          <path
            d="M715.8 493.5L335 165.1c-14.2-12.2-35-1.2-35 18.5v656.8c0 19.7 20.8 30.7 35 18.5l380.8-328.4c10.9-9.4 10.9-27.6 0-37z"
          />
        </svg>
      </span>
      Nghe bài hát
    </div>
  </div>
</div>
`;
